packages/connectkit/src/utils/trpc.ts

```diff
@@ -1,14 +1,18 @@
-// TODO: re-enable for dev only in rollup.
-// import { AppRouter } from "@daimo/pay-api";
-import { createTRPCClient, httpBatchLink } from "@trpc/client";
+import type { AppRouter } from "@daimo/pay-api";
```

Comment:
if we use `import type` + don't mention `pay-api` as  dependency in the package file, I think we can strongly type the API again.

Reply:
does `npm run build/dev` without the `as any` in the function generate folders without the API deps? if so feel free to drop the as any

Reply:
fixed build; output is clean now, no `pay-api` types.